app/views/episodes/_form_tags.html.erb

```diff
@@ -6,16 +6,18 @@
 
     <div class="card-body">
       <div class="form-floating input-group">
-        <%= form.tag_select :categories, episode.categories, {}, data: {allow_new: true, allow_clear: true, category_target: "field"} %>
+        <%= form.tag_select :categories, episode.categories, {}, data: {allow_new: policy(episode).update?, allow_clear: policy(episode).update?, category_target: "field"} %>
```

Comment:
technically didn't need to also add the policy to the `allow_new` param, but it seemed like an opportunity to keep it consistent.